app/src/main/java/com/example/foodjuju/ui/DTO/FoodJuJuDTO.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.foodjuju.ui.DTO
+
+class FoodJuJuDTO {
```

Comment:
An empty class is not adding much value...  it's clutter.  Only completed features should be merged to master.

Also, if it is a DTO, shouldn't it be a data class?